integration-tests/collectors/run-test.sh

```diff
@@ -0,0 +1,300 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+if [ -z "$GITHUB_TOKEN" ] ; then
+  echo "error: missing env var GH_TOKEN"
```

Comment:
it should be `GITHUB_TOKEN`, not `GH_TOKEN` in the message.

Reply:
fixed